src/terms.jl

```diff
@@ -378,7 +418,7 @@ end
 
 Base.show(io::IO, t::InterceptTerm{H}) where {H} = print(io, H ? "1" : "0")
 
-Base.show(io::IO, t::ContinuousTerm) = print(io, t.sym)
+Base.show(io::IO, t::ContinuousTerm) = print(io, t.sym,"\n" , t.mean,"\n" , t.var, "\n" ,t.min,"\n" , t.max)
```

Comment:
this is irrelevant for checking equality and breaks existing behavior.  It also causes the tests to fail.  As a general suggestion, make sure that the test suite runs (by doing `Pkg.test()` or running `test StatsModels` in pkg mode) before you submit a PR.